draft-ietf-sidrops-aspa-profile.xml

```diff
@@ -110,19 +110,20 @@
           <t>further propagates in all directions (towards providers, lateral peers, and customers) any BGP Updates that the customer may send.</t>
         </list>
         The digitally signed Autonomous System Provider Authorization (ASPA) object described in this document provides the above-mentioned authorization mechanism.
-        See <xref target="I-D.ietf-sidrops-aspa-verification"/> for a specification how to use Validated ASPA Payloads (VAPs) to filter BGP UPDATE messages.
       </t>
       <t>
         An ASPA object is a cryptographically verifiable attestation signed by the holder of an Autonomous System identifier (hereafter called the "Customer AS", or CAS).
         An ASPA contains a list of one or more ASes, each listing meaning the listed AS is authorized to act as Provider network.
-        When the CAS has multiple Providers, all Provider ASes that provide service to the CAS are listed in the ASPA, including any non-transparent Internet Exchange Point (IXP) Route Server (RS) ASes.
-        The common case for Route Servers (RS) at Internet Exchange Points is to operate transparently (see Section 2.2.2.1 <xref target="RFC7947"/>), thus usually, the ASNs of IXP Route Servers are not listed as PAS in ASPAs.
-	The detailed ASPA registration recommendations are provided in <xref target="I-D.ietf-sidrops-aspa-verification"/>. 
-      </t>
+        When the CAS has multiple Providers, all Provider ASes are listed in the ASPA, including any non-transparent Internet Exchange Point (IXP) Route Server (RS) ASes.
+        The common case for RS ASes at IXPs is to operate transparently (see Section 2.2.2.1 <xref target="RFC7947"/>), and in those instances the ASNs of IXP Route Servers are not listed as PAS in ASPAs.
+	  </t>
       <t>
-        The ASPA content type definition conforms to the <xref target="RFC6488"/> template for RPKI signed objects.
+		The BGP Roles that an Autonomous System (AS) may have in its peering relationships with eBGP neighbors are discussed in <xref target="I-D.ietf-sidrops-aspa-verification"/>.
```

Comment:
Why isn’t “The BGP Roles…” starting after the same number of whitespaces as the old line “The ASPA content type…” ?

This is an example of misalignment: normally each level of the XML starts two whitespaces deeper than the previous level.